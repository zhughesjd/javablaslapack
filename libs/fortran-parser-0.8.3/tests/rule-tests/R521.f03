! Testing asynchronous-stmt, R521
10  asynchronous a
ASYNCHRONOUS :: B, C, scalar
end
