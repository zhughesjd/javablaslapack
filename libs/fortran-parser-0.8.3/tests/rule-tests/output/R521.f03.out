R1101:main-program__begin:
R 102:generic-name-list__begin:
R 102:generic-name-list-part: id=[@7,51:51='a',<206>,2:17]
R 102:generic-name-list: count=1
R 447:asynchronous-stmt: label=[@2,34:35='10',<10>,2:0]
R 102:generic-name-list__begin:
R 102:generic-name-list-part: id=[@13,69:69='B',<206>,3:16]
R 102:generic-name-list-part: id=[@16,72:72='C',<206>,3:19]
R 102:generic-name-list-part: id=[@19,75:80='scalar',<206>,3:22]
R 102:generic-name-list: count=3
R 447:asynchronous-stmt:
R 204:specification-part: numUseStmts=0 numImportStmts=0 numDeclConstructs=2
R1103:end-program-stmt: id=null
R1101:main-program: hasProgramStmt=false hasExecutionPart=false hasInternalSubprogramPart=false
